### ISSUE TITLE
Narrator + Edge in Scan Mode doesn't announce series while navigating through data points

### ISSUE THREAD

**Issue by JllyGrnGiant** [highcharts/highcharts#9412]
#### Expected behaviour
Narrator announces the series associated with the highlighted data point
(e.g. the first data point should say something like "Image 1, Jaws, January 2009, 74")

#### Actual behaviour
Narrator announces "Image 1, January 2009, 74" (missing "Jaws")

#### Live demo with steps to reproduce
Highcharts 6.2.0 at the time of creating this issue.
https://www.highcharts.com/samples/highcharts/accessibility/accessible-line/

1. Navigate to demo
2. Turn on Narrator
3. Tab to chart
4. Press down/up arrows repeatedly to navigate through chart data points

#### Product version
Highcharts 6.2.0

#### Affected browser(s)
Edge

**Comment by KacperMadej** [comment]
Hi @JllyGrnGiant 

Thank you for reporting about the problem.

@oysteinmoseng - accessibility module issue

**Comment by oysteinmoseng** [comment]
This can be configured by setting the point format (https://api.highcharts.com/highcharts/accessibility.pointDescriptionFormatter or https://api.highcharts.com/highcharts/plotOptions.series.pointDescriptionFormatter), but we will also add the series to the default formatter since it makes sense to include it.

**Comment by emrusso** [comment]
Hi @oysteinmoseng

> we will also add the series to the default formatter

When is this change taking place?

**Comment by oysteinmoseng** [comment]
Hi @emrusso, this will be part of our v7.1 release, scheduled for end of March, early April.
